analyzers/src/SonarAnalyzer.CFG/ShimLayer/TypeKindEx.cs

```diff
@@ -0,0 +1,30 @@
+﻿/*
```

Comment:
As it was us adding this file to the ShimLayer, I am not sure which license header we should use. This one or the other one that is used in all the other ShimLayer files? Maybe @pavel-mikula-sonarsource has an answer for us?